CharacterCreation/06_01_Systemized_Skills.txt

```diff
@@ -1,68 +1,125 @@
+do contribute to the maximum of 60 points in a skill.
+
+	Characters are considered to be proficient with certain skills. When a
+character is proficient in a skill, each skill point they spend in that skill
+upon leveling up increases that skill by two instead of one.
+
+===== Skill Checks and Critical Successes =====
+	If you roll a critical success on a skill check, add 10 to your total roll.
+For an extra critical, add 20.
+
+===== "General" Skills =====
+	Some skill groups such as Knowledge, Weapons, and Vehicles allow
+characters to learn the skill both in general and in specific cases. If the
+character does not know the specific skill in a group that applies to a situation,
+the player can instead add half of the character's general skill in that skill
+group (rounded down). For example, if a GM asks for a roll in Knowledge: History,
+but the character only has 10 skill in Knowledge: General, the player can add 5
+from Knowledge: General to the Knowledge: History roll. However, if the  GM asks
+for a roll in Knowledge: General, the player can all all 10 points from
+Knowledge: General to the roll.
 
+===== Skills in Compound X =====
+	This list of some of the most commonly-used skills in Compound X is
+arranged based on the stats that are usually associated with the skill. Your GM
+may decide that a different stat applies to any given instance. In addition,
+this is not an exhaustive list of skills, merely the skills called for most
+often. Talk to your GM if you would like your character to learn a more specific
+case of one of these skills or a skill not listed below.
+
+== Charisma ==
+Deception:
+	To tell a convincing lie
+Insight:
+	To tell truth and lies apart
+Performance:
+	"Roll for interpretive dance"
+Persuasion:
+	To tell the truth, share an opinion, or change someone's mind
+
+== Dexterity ==
+Acrobatics:
+	"PARKOUR!"
+Sleight of Hand:
+	To pick pockets, pull off card tricks, and cause other mayhem
+Stealth:
+	To move quietly and escape notice
 Vehicles:
-* GENERAL*
+	For checks to pilot vehicles
+* General
 * Flight
 * Ground
 
-Weapons:
-* GENERAL*
-* Melee
-* Long Rifle
-* SMG
-* Shotgun
-* General
-* Pistol
-* Thrown
-* Heavy
-
+== Fortitude ==
+Intimidation:
+	To strike fear into the hearts of your enemies
+	
+== Intelligence ==
+Hacking:
+	To pwn computer systems
+Medicine:
+	To repair life forms
+Repair:
+	To repair machines
+Jury-rigging:
+	To throw together a solution using the materials on hand
+Navigation:
+	For plotting routes through space or across land
+Survival:
+	To make do far from civilization with limited technological help
 Knowledge:
-* GENERAL*
-* Gravity Manipulation
-* Energy Manipulation
-* Hard Light
+	For when a character's knowledge differs from the player's
+* General
+* Computers
+* Geography
+* High Tech
 * History
 * Nanites
-* Plasma
-* Computer Systems
+* Religion
+* Underworld
 * Xenology
+Weapons:
+	To properly reload, care for and repair a weapon

```

Comment:
nonblocking: I'd like a little more detail here. I'd be willing to add it myself. I want to explicitly call out that weapon skill helps with reloading, clearing jams, adding attachments in the field and goes along with certain weapon feats, but very specifically does not change accuracy rolls. I think this is the place to neutralize a very common confusion point.

Reply:
and you did a very good job creating the context for that opportunity to destroy confusion!

Reply:
Specified that the skill doesn't affect accuracy.